tests/bluetooth/tester/src/btp_gatt.c

```diff
@@ -2139,6 +2140,71 @@ static uint8_t notify_mult(const void *cmd, uint16_t cmd_len,
 }
 #endif /* CONFIG_BT_GATT_NOTIFY_MULTIPLE */
 
+static uint8_t get_handle_from_uuid(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_gatt_get_handle_from_uuid_cmd *cp = cmd;
+	struct btp_gatt_get_handle_from_uuid_rp *rp = rsp;
+	struct bt_uuid search_uuid;
+
+	if (btp2bt_uuid(cp->uuid, cp->uuid_length, &search_uuid)) {
+		return BTP_STATUS_FAILED;
+	}
+
+	__maybe_unused char uuid_str[BT_UUID_STR_LEN];
+
+	bt_uuid_to_str(&search_uuid, uuid_str, sizeof(uuid_str));
+
+	LOG_DBG("Searching handle for UUID %s", uuid_str);
+
+	for (int i = 0; i < attr_count; i++) {
```

Comment:
```suggestion
	for (uint8_t i = 0; i < attr_count; i++) {
```

Nitpick

Reply:
`size_t` is probably preferred for for-loops, though there is no rule regarding using it in Zephyr and we use both `int` and `size_t`. `uint8_t` should never be used in for-loops as it doesn't help to optimize code and may hit in a long run if for loop iterates through more than 256 entries.

Reply:
> I think that's a unbased claim, without knowing what compiler or board is being used. If the compiler ends up treating both value as 32-bit values (int), then there's an increase in stack size compared to both being treated as 8-bit values, right?

ARM Cortex-M family uses 32-bit registers and stack is 32-bit aligned. uint8_t doesn't gain here anything, but most likely will make compiler produce more instructions to handle uint8_t properly.